tensorbay/cli/auth.py

```diff
@@ -132,3 +134,20 @@ def _check_args_and_options(arg1: str, arg2: str, get: bool, unset: bool, is_all
 def _echo_formatted_profile(name: str, value: str) -> None:
     formatted_value = indent(value, INDENT)
     click.echo(f"{name} = {formatted_value}\n")
+
+
+def _check_auth(arg1: str, arg2: str, profile_name: str) -> None:
+    if arg2:
+        arg1, arg2 = arg2, arg1
+    gas_client = get_gas(arg1, arg2, profile_name)
+    try:
+        userinfo = gas_client.get_user()
+    except ResponseSystemError:
+        error(f"{arg1} is not a valid AccessKey")
+    else:
+        message = f"\nSuccessfully set authentication info of {userinfo.name}"
+        if userinfo.team:
+            message += f" in {click.style(userinfo.team.name, bold=True, fg='blue')} team"
```

Comment:
use `join` instead of `str.__iadd__`
python `str` is immutable, `str.__iadd__` will create new string, which is slow and cost more memory